src/System.Diagnostics.TraceSource/src/System/Diagnostics/TraceInternal.cs

```diff
@@ -371,6 +355,7 @@ public static void TraceEvent(TraceEventType eventType, int id, string format, p
                     {
                         foreach (TraceListener listener in Listeners)
                         {
+                            listener.IndentLevel = t_indentLevel;
```

Comment:
A note about [TraceListener.IndentLevel](https://msdn.microsoft.com/en-us/library/system.diagnostics.tracelistener.indentlevel(v=vs.110).aspx). Given that this is now controlled by the Indent / Unindent calls at per thread level, should this be anymore provided on the TraceListener interface? Or then, should it be a 'get' property only?